frontend/src/data/variables/AcsPopulationProvider.test.ts

```diff
@@ -187,6 +192,56 @@ describe("AcsPopulationProvider", () => {
     );
   });
 
+  test("State and Race Breakdown with standard race filter", async () => {
+    const rawData = [
+      stateRow(AL, RACE, TOTAL, 2),
+      stateRow(AL, RACE, ASIAN, 2),
+      stateRow(NC, RACE, TOTAL, 20),
+      stateRow(NC, RACE, ASIAN, 5),
+      stateRow(NC, RACE, WHITE, 15),
+      // Non-standard, will be excluded from the non-standard filter
+      stateRow(NC, RACE, "White", 17),
```

Comment:
Maybe we should rename WHITE to WHITE_NH and create WHITE as "White"?

Reply:
Done, and added a TODO to add constants for other values